guides/ipynb/keras_cv/coco_metrics.ipynb

```diff
@@ -44,7 +44,7 @@
     "`bounding_box_format` parameter.  This parameter is used to tell the components what\n",
     "format your bounding boxes are in.  While this guide uses the `xyxy` format, a full\n",
     "list of supported formats is available in\n",
-    "[the bounding_box API documentation](/api/keras_cv/bounding_box/formats).\n",
+    "[the bounding_box API documentation](https://keras.io/api/keras_cv/bounding_box/formats/).\n",
```

Comment:
Please:
- Remove the `https://keras.io` prefix
- Apply the change to the `py` and `md` files as well.

Reply:
Hi @fchollet ! The link is not working . If I remove the keras.io prefix.

Reply:
I guess you're trying to follow the link when hosting the notebook outside of the Keras website, e.g. on Colab?

Reply:
Yeah. You are correct. It is working fine within website but throwing error when opened from  Github source code.

Reply:
Shall i close this PR then?

Reply:
I think it's a legit concern. We can merge the PR. Please also update the `md` file.

Reply:
Ok, I see it was updated already. Good to go!